api/config/infra_proxy/config_request.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package chef.automate.domain.infra_proxy;
+option go_package = "github.com/chef/automate/api/config/infra_proxy";
+
+import "api/config/shared/global.proto";
+import "api/config/shared/tls.proto";
+import "components/automate-grpc/protoc-gen-a2-config/api/a2conf/annotations.proto";
+import "google/protobuf/wrappers.proto";
+
+message ConfigRequest {
+	option (chef.automate.api.service_config) = {name: "infra-proxy-service"};
+
+	reserved 1 to 2;
+	V1 v1 = 3;
```

Comment:
[minor]  Since this is a brand new proto, no need to start with fields 1 and 2 as reserved, this can be `V1 v1 =1` and you can removed `reserved 1 to 2;` above (this will require rebuilding config protos.